02-oop-inheritance/patterns-1.1.html

```diff
@@ -6,15 +6,17 @@
     <meta charset="utf-8">
     <title>Patterns and OOP in JavaScript</title>
 
-    <script type="text/javascript" src="patterns-1.1.js"> </script>
+    <script type="text/javascript" src="C:\Users\juan.almeida\startup\lib\patterns-1.1.js"> </script>
   </head>
 
   <body>
 
     <div class="container">
-
+        <label>Title </label>
```

Comment:
awesome! but remember to add the *for* attribute (i.e. ``<label for"movieTitle">``)